tests/test_bs.py

```diff
@@ -184,9 +187,12 @@ def _setup_bootstrap_in_namespace(name_space):
                                  container_specs=cspec)
 
         bootstrap_deployment_info.deployment_name = resp.metadata._name
-        namespaced_pods = client.CoreV1Api().list_namespaced_pod(namespace=name_space).items
-        bootstrap_pod_json = next(filter(lambda i: i.metadata.name.startswith(bootstrap_deployment_info.deployment_name),
-                                         namespaced_pods))
+        # The tests assume we deploy only 1 bootstrap
```

Comment:
if we can not assume this it'll be better. I know that is the situation today but soon we'll want to test with multiple boot nodes.

Reply:
Supporting many bootstraps will require many changes. This is not the scope for this PR.
This is why I added the comment here